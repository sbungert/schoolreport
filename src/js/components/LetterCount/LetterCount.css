/* Letter Count */
/* ------------ */

.LetterCount {
  display: inline-block;
  color: var(--col-darken-3);
  padding-left: 10px;
}