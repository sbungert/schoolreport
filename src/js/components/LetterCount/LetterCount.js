// @flow

import React from 'react';
import './LetterCount.css';

export type Props = {
  count: string,
};

const LetterCount = (props: Props) => {
  return <i className="LetterCount">{props.count}</i>;
};

export default LetterCount;
