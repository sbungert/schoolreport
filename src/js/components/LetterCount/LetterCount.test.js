// @flow

import React from 'react';
import ReactDOM from 'react-dom';
import LetterCount from './LetterCount';
import type { Props } from './LetterCount';

describe('<LetterCount />', () => {
  const props: Props = {
    count: '5',
  };

  test('Renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<LetterCount {...props} />, div);
  });
});
