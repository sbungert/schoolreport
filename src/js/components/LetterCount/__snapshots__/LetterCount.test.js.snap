// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LetterCount /> Props render correctly 1`] = `
<i
  className="LetterCount"
>
  5
</i>
`;
